codegen/src/main/twirl/templates/PlayJava/AkkaGrpcClientModule.scala.txt

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ *@
```

Comment:
rather I would say such generated files should contain info like "generated ...."?

Reply:
Whoops my bad, that's just comment here...

How about adding the // generated... comment though anyway or should we do this together in all files later?

Reply:
Yeah, lets' do that separately, I think we have a ticket for it already

Reply:
(I mean a huge "GENERATED DO NOT EDIT" at the top of the file, not as scaladoc on the classes, same as protobuf does)

Reply:
#273